pyanaconda/payload/livepayload.py

```diff
@@ -294,7 +294,7 @@ def _setup_file_image(self):
         if not os.path.exists(self.data.method.url[7:]):
             return "file does not exist: %s" % self.data.method.url
 
-        self._min_size = os.stat(self.data.method.url[7:])[stat.ST_SIZE] * 3
+        self._min_size = os.stat(self.data.method.url[7:]).st_blocks * 512 * 3
```

Comment:
Any chance here block size can be different than 512, for example 4k?

Reply:
st_blocks is defined as "Number of 512B blocks allocated" (from man 2 stat)